pallets/inflation-manager/src/lib.rs

```diff
@@ -300,6 +300,19 @@ pub mod pallet {
 
 			Ok(().into())
 		}
+
+		#[pallet::call_index(3)]
+		#[pallet::weight(T::WeightInfo::set_block_reward())]
+		pub fn set_block_reward(
+			origin: OriginFor<T>,
+			reward: Balance,
+		) -> DispatchResultWithPostInfo {
+			ensure_root(origin)?;
+
+			BlockRewards::<T>::put(reward);
```

Comment:
If we change block rewards this way, our total supply at end of each progressive year will be different from the initial tokenomics design. I think we should document these newer extrinsics. I'll create a ticket for that and come back to it later.